packages/ember-glimmer/tests/utils/abstract-test-case.js

```diff
@@ -48,7 +48,7 @@ export function moduleFor(description, TestClass, ...mixins) {
 
   QUnit.module(`[${packageName}] ${descriptionWithoutPackagePrefix}`, {
     setup() {
-      context = new TestClass();
+      context = new TestClass(assert);
```

Comment:
By the way, if we decide to pass args to the class, we would have to update all the constructors to call super with the args, which would be somewhat of a project (probably takes 15 minutes or so)